### DISCUSSION TITLE
Modify "year" citation key formula

### DISCUSSION THREAD

**Discussion by AlexandraGloria** [retorquere/zotero-better-bibtex#2915] [Q&A] [answered]
Hello,
I have the citation key formula set to "auth + year", which gives me the output e.g. Author2002 for my citation key. Is there a way to modify the "year" portion in the formula so the year in my citation key is set to the last two digits of the year (i.e., Author02)?
Cheers!

**Comment by retorquere** [comment, marked as answer]
`shortyear`.

**Reply by AlexandraGloria** [reply]
Thanks!

**Comment by fabiez** [comment]
Hi all,
How to obtain a citation that looks as follows: "Lastname, Firstname_Title_Year"?
So far I use `{{authorString}}|{{title}}|{{year}}`, but this places the first name before the second.
Many thanks
Fabio

**Reply by retorquere** [reply]
citation keys cannot have spaces.

**Reply by retorquere** [reply]
`{{authorString}}|{{title}}|{{year}}` is not a valid BBT citation key formula (it is neither the legacy format nor the new `+` format, so I'm not sure what your point of departure is. BBT will not allow you to enter this.

**Reply by retorquere** [reply]
Is this perhaps the Zotero file rename syntax?
